config/environments/production.rb

```diff
@@ -14,7 +14,7 @@
   config.eager_load = true
 
   # Full error reports are disabled and caching is turned on.
-  config.consider_all_requests_local       = false
+  config.consider_all_requests_local       = true
```

Comment:
It's probably a good idea to also update the comment introducing this section to keep them in sync…

Reply:
Yep, I've changed this